index.html

```diff
@@ -89,126 +89,47 @@ <h2 class="headline-connect">LET'S CONNECT</h2>
+          CANOPY <span class="dot tag"> Back End dev </span>
+          <span class="dot tag">2015</span>
+        </h4>
         <div class="content1">
-          <h3 class="work-title">Tonic</h3>
-          <h4>
-            CANOPY <span class="dot tag"> Back End dev </span>
-            <span class="dot tag">2015</span>
-          </h4>
-          <p class="title-description">
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required.
-          </p>
-          <ul class="stack-group">
-            <li class="stack">html</li>
-            <li class="stack">css</li>
-            <li class="stack">javascript</li>
-          </ul>
-          <button type="button" class="btn">See Project</button>
-        </div>
-      </div>
-
-      <!-- Work Section -->
-
-      <div class="work-info4">
-        <div class="image-container">
-          <img
-            src="./images/work-img4.png"
-            alt="multi-post work content2"
-            class="work-img"
-          />
-        </div>
-        <div class="content1">
-          <h3 class="work-title">
-            Multi-Post <br />
-            Stories
-          </h3>
-          <h4>
-            CANOPY <span class="dot tag"> Back End dev </span>
-            <span class="dot tag">2015</span>
-          </h4>
-          <p class="title-description">
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required.
-          </p>
-          <ul class="stack-group">
-            <li class="stack">html</li>
-            <li class="stack">css</li>
-            <li class="stack">javascript</li>
-          </ul>
-          <button type="button" class="btn">See Project</button>
+          <div class="image-contain fix">
+            <img
+              src="./images/work-img4.png"
+              alt="multi-post work content2"
+              class="work-img"
+              id="modal-img"
+            />
+          </div>
+          <div class="stack-modal">
+            <p class="title-description" id="modal-desc">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. A daily selection of privately personalized
+              reads; no accounts or sign-ups required. A daily selection of
+              privately personalized reads; no accounts or sign-ups required.
+            </p>
+            <div>
+              <ul id="group-list" class="stack-group"></ul>
+              <div class="stack-group">
+                <a class="btn-modal" id="modal-see-live"
+                  >See live<span
+                    ><img src="./images/export.png" alt="export" /></span
+                ></a>
+                <a class="btn-modal" id="modal-see-source"
+                  >See Source<span
+                    ><img src="./images/github.png" alt="github" /></span
+                ></a>
+              </div>
+            </div>
+          </div>
```

Comment:
- [ ] I just noticed from the project requirements that the **Popup** code is also supposed to be defined in your javascript file(e.g.:you can only define its **container** tag from the HTML file and then use Dom manipulation for its content(**as you for the cards at line 92**)).

- [ ] Once you fix the above issue your Popup design sticks to the one from Figma as required for the project(e.g.: width of the popup's content block, space around, background opacity, blur effect,...)
### Figma Design
![Screenshot from 2022-07-21 18-05-11](https://user-images.githubusercontent.com/83042742/180265142-2ec31bb1-1455-464b-91df-dddd8c1e47cf.png)

### Your Design
![Screenshot from 2022-07-21 18-26-17](https://user-images.githubusercontent.com/83042742/180265716-36a61238-3ad5-4fcc-a425-f0596d8d9b0c.png)

- [ ] Also Kindly make sure the popup is responsive, it fits mobile and desktop sizes properly. Below are some overlapping issues in the Desktop version of some sizes (I think this happen because you gave the popup content block in the white background a fixed **width** and **height**):
![Screenshot from 2022-07-21 18-03-57](https://user-images.githubusercontent.com/83042742/180266191-0130e7ed-a72b-4794-af6f-0a065b1233a9.png)

Reply:
Please i have tried implementing the pop-up in JavaScript and it doesn't work when the button is clicked. i have sought ways to fix it and i still cant. Can you offer a solution?

Reply:
@tresorsawasawa Please i have tried implementing the pop-up in JavaScript and it doesn't work when the button is clicked. i have sought ways to fix it and i still cant. Can you offer a solution?

Reply:
@tresorsawasawa please be aware that the image from the figma design for pop-up is not the same size with the images that is used hence it cant fill the width of the container. When stretched to fill the container it looks ugly to the eyes.
The responsiveness has been fixed.

Reply:
The modal is dynamic too.

Reply:
Answered on Slack